frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -41,10 +67,16 @@ const TableFilter = (props) => {
     updateSelectedFilter,
   } = props;
 
-  const category =
-    filterData?.length > 0 &&
-    categoryFilterValue &&
-    filterData.filter((item) => item.name === categoryFilterValue)?.[0]?.key;
+  const getFilterID = (name) => {
```

Comment:
Here, the function retrieves the current filter category, so its name should be appropriate.